Extensions/DraggableBehavior/draggableruntimebehavior.ts

```diff
@@ -119,7 +119,15 @@ namespace gdjs {
    * Handle the dragging
    */
   abstract class DraggableManager {
+    /**
+     * The object has left its original position.
+     * When true, the search for the best object to drag has ended.
+     */
     protected _draggingSomething = false;
+    /**
+     * The behavior of the object that is best one found so far.
```

Comment:
nitpicking: 
```suggestion
     * The behavior of the object that is being dragged and that is the best one (i.e: highest Z order) found.
```

Reply:
I wanted to explain that the value can still change until all the behaviors doStepPreEvents are called, but it was more confusing than anything.